swift-sdk/IterableMessaging.swift

```diff
@@ -121,47 +156,57 @@ public final class IterableInAppMessage : NSObject {
     /// the campaign id for this message
     public let campaignId: String
     
-    /// the in-app type
-    public let inAppType: IterableInAppType
-    
     /// when to trigger this in-app
     public let trigger: IterableInAppTrigger
     
     /// when to expire this in-app, nil means do not expire
     public let expiresAt: Date?
     
-    /// The content of the inApp message
+    /// The content of the inbox message
```

Comment:
Not necessarily inbox, `IterableInAppContent` is common for any in-app messages.